### ISSUE TITLE
Monitor config with Y offset

### ISSUE THREAD

**Issue by TheBlueOompaLoompa** [OpenTabletDriver/OpenTabletDriver#2924]
### Steps to Reproduce

1. Setup your monitors to be below Y=0
2. Set the display area in opentabletdriver to be on said display below y=0
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/53215039/0ec2ac39-40dc-4bd2-96cb-24ec7e74587b)

### Expected Behavior

The cursor should be positioned like normal.

### Observed Behavior

The cursor snaps to the bottom edge of the tablet screen and refuses to move up, but is offset strangely in the X-axis. I can't show this behavior as the cursor is moving to the edge where it can't be seen.

### Tablet Device

Huion Kamvas 12

### Diagnostic Information

[diag.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/12566420/diag.txt)

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

**Comment by gonX** [comment]
Ah, I think I reproduced it, but only after changing my resolution without restarting the daemon. This smells too much like a duplicate of #1143
